common.blocks/sorting/sorting.js

```diff
@@ -0,0 +1,38 @@
+modules.define('sorting', ['i-bem__dom', 'select', 'jquery'], function(provide, BEMDOM, Select, $) {
+  provide(BEMDOM.decl(this.name, {
+    onSetMod: {
+      js: {
+        'inited': function() {
+          Select.on(this.elem('select'), 'change', this._onSortingSelectChanged, this);
+        }
+      }
+    },
+    _onSortingSelectChanged: function(e) {
+      var queryString = this._getQueryString(e.target.getVal()),
+      contentBlock = this.findBlockOutside('content'),
+      contentLeftSpinBlock = contentBlock.findBlockInside('content__spin').findBlockOn('spin'),
+      issuesBlock = contentBlock.findBlockInside('issues');
+
+      issuesBlock.setMod('loading');
+      contentLeftSpinBlock.setMod('visible');
+
+      $.get(`/?${queryString}`)
+      .then(function(data) {
+        issuesBlock.delMod('loading');
+        BEMDOM.replace(issuesBlock.domElem, data);
+        contentLeftSpinBlock.delMod('visible');
+      });
+    },
+    _getQueryString: function(val) {
+      switch (val) {
```

Comment:
oh, actually there's no need in this function at all, just move the values to https://github.com/tadatuta/bem-forum/pull/21/files#diff-9f325be26d59702b218b09327cffecc2R13 instead of numbers